scripts/setup.sh

```diff
@@ -6,6 +6,7 @@ pushd $(git rev-parse --show-toplevel) > /dev/null
 
 cargo install sqlx-cli --version 0.7.4  --no-default-features --features postgres,rustls
 cargo sqlx database create
+DATABASE_URL="${DATABASE_URL}_test" cargo sqlx database create
```

Comment:
Oh we don't need this! We do it before we run tests in Playwright now, will remove